src/main/io/displayport_crsf.c

```diff
@@ -84,7 +84,7 @@ static int crsfWriteString(displayPort_t *displayPort, uint8_t col, uint8_t row,
     if (row >= crsfScreen.rows || col >= crsfScreen.cols) {
         return 0;
     }
-    const size_t truncLen = MIN((int)strlen(s), crsfScreen.cols-col);  // truncate at colCount
+    const size_t truncLen = MIN(strlen(s), (size_t)(crsfScreen.cols - col));  // truncate at colCount
```

Comment:
Don't know if we need the (size_t) cast instead
```suggestion
    const size_t truncLen = MIN((size_t)strlen(s), (size_t)(crsfScreen.cols - col));  // truncate at colCount
```

Reply:
Thank you @daleckystepan It's building now!!! Make test also works!

Reply:
I don't think it's necessary. AFAIK strlen already returns size_t.